Documentation/PageTsconfig/Rte.rst

```diff
@@ -95,262 +102,179 @@ Summary:
     # Override preset for field "description" in table "tt_address"
     RTE.config.tt_address.description.preset = custom_preset_fancy
 
+..  _pageTsRte-properties:
 
 Properties
 ==========
 
-.. contents::
-   :depth: 2
-   :local:
+..  contents::
+    :depth: 2
+    :local:
 
-.. index:: RTE; disable
+..  index:: RTE; disable
 
 disabled
 --------
 
-:aspect:`Datatype`
-    boolean
+..  confval:: disabled
+    :name: rte-disabled
+    :type: boolean
 
-:aspect:`Description`
     If set, the editor is disabled. This option is evaluated in :php:`\TYPO3\CMS\Backend\Form\FormEngine`
     where it determines whether the RTE is rendered or not. Note that a backend user can also ultimately
     disable RTE's in his user settings.
 
-.. index::
-   RTE; Configuration
-   RTE; config
+..  index::
+    RTE; Configuration
+    RTE; config
 
 
 buttons
 -------
 
-
-
-.. _buttons-link-options-removeitems:
+..  _buttons-link-options-removeitems:
 
 buttons.link.options.removeItems
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. container:: table-row
-
-   Property
-         buttons.link.options.removeItems
-
-   Data type
-         list of strings
-
-   Description
-         List of tab items to remove from the dialog of the link button.
-         Possible tab items are: page, file, url, email, folder, telephone.
-
-         Note: More tabs may be provided by extensions.
+..  confval:: buttons.link.options.removeItems
+    :name: rte-buttons-link-options-removeItems
+    :type: list of strings
 
+    List of tab items to remove from the dialog of the link button.
+    Possible tab items are: page, file, url, email, folder, telephone.
 
+    Note: More tabs may be provided by extensions.
 
-.. _buttons-link-targetselector-disabled:
+..  _buttons-link-targetselector-disabled:
 
 buttons.link.targetSelector.disabled
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. container:: table-row
-
-   Property
-         buttons.link.targetSelector.disabled
-
-   Data type
-         boolean
-
-   Description
-         If set, the selection of link target is removed from the link
-         insertion/update dialog.
-
-         Default : 0
+..  confval:: buttons.link.targetSelector.disabled
+    :name: rte-buttons-link-targetselector-disabled
+    :type: boolean
+    :Default: false
```

Comment:
```suggestion
    :Default: 0
```

This is page TSconfig, not PHP. I assume, "false" may be interpreted as true as it is a non-empty string.